handlers/api.go

```diff
@@ -2147,7 +2148,25 @@ func ApiGraffitiwall(w http.ResponseWriter, r *http.Request) {
 	if slotQuery < 10000 {
 		slotQuery = 10000
 	}
-	rows, err := db.ReaderDb.Query("SELECT x, y, color, slot, validator FROM graffitiwall WHERE slot <= $1 AND slot >= $2 ORDER BY slot desc", slotQuery, slotQuery-10000)
+
+	startPxl := uint64(0)
+	endPxl := uint64(999)
```

Comment:
**Minor:** These sould have more meaningful names like `defaultStartPxl` and `defaultEndPxl`.

Reply:
I had those at first but didn't quite like them because I also use **endPxl** as an upper boundary but by now I probably like those names more than startPxl/endPxl so I changed it.